eslint.config.mjs

```diff
@@ -192,6 +188,7 @@ export default tseslint.config([
   },
   globalIgnores([
     '.venv/*',
+    '.yarn/*',
```

Comment:
ESLint was spending many seconds linting the `.yarn/releases/yarn-4.8.1.cjs` file; AFAICT it didn't do that in v8. I'm guessing this was ignored by default in v8? It wasn't in our previous `.eslintignore` file 🤷